src/components/dashboard/FaceVerification/screens/IntroScreen.js

```diff
@@ -184,27 +185,27 @@ const getStylesFromProps = ({ theme }) => ({
     width: '100%',
   },
   mainTitle: {
-    marginTop: getDesignRelativeHeight(isBrowser ? 30 : 15),
+    marginTop: getDesignRelativeHeight(isLargeDevice ? 30 : 10),
```

Comment:
Why do we need to replace 'isBrowser' to 'isLargeDevice'?
I used an 'isBrowser' to apply style just for desktop view and the rest for phones.

Reply:
mmm for a strange reason 'isBrowser' return true on native and suppose that is not wanted.

Reply:
1. fix isBrowser across the app
2. we have alternatives like isMobile

Reply:
I think that 'isWeb' is a good solution for this case.
Following the documentation from mobile-device-detect, is Browser returns true if device type is browser.
@yaroslav-fedyshyn-nordwhale  the use of 'isBrowser' is for a particular reason? or could we use isWeb in this case?

Reply:
As I mentioned before - I used 'isBrowser' to apply styles just for desktops.
If 'isBrowser' is returning true for native apps which is not good for you - then better to change 'isBrowser' to the !isMobile and be sure it is applied only for desktop.